### PR TITLE
[Yaml] set arguments depending on the PHP version

### PR DESCRIPTION
**Pull request by xabbuh**
| Q             | A
| ------------- | ---
| Branch?       | 3.1
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | n/a
| License       | MIT
| Doc PR        | n/a

### CODE REVIEW

**Comment by nicolas-grekas** [comment]
So, regarding https://github.com/symfony/symfony/pull/20291#discussion_r86390433
you propose: wait and see?

**Comment by xabbuh** [comment]
Yeah, I think it doesn't hurt to wait what HHVM will do in the future. We can still adapt the test then if needed.

File: src/Symfony/Component/Yaml/Tests/InlineTest.php [hunk 1]
```diff
@@ -507,7 +507,12 @@ public function testParseTimestampAsDateTimeObject($yaml, $year, $month, $day, $
         $expected = new \DateTime($yaml);
         $expected->setTimeZone(new \DateTimeZone('UTC'));
         $expected->setDate($year, $month, $day);
-        @$expected->setTime($hour, $minute, $second, 1000000 * ($second - (int) $second));
+
+        if (PHP_VERSION_ID >= 70100) {
```
Comment (iltar) [inline src/Symfony/Component/Yaml/Tests/InlineTest.php]: Will this work for different HHVM versions?

Comment (xabbuh) [inline src/Symfony/Component/Yaml/Tests/InlineTest.php]: Depends on which PHP version they claim to be compatible with.

Comment (nicolas-grekas) [inline src/Symfony/Component/Yaml/Tests/InlineTest.php]: This is a test case, we don't care...

**Comment by fabpot** [comment]
Thank you @xabbuh.

**Pull request merged**